examples/domainmodel/test/domainmodel-cli.test.ts

```diff
@@ -0,0 +1,52 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import path from 'path';
+import fs from 'fs';
+import { exec, ExecException } from 'child_process';
+
+describe("Test the domainmodel CLI", () => {
+    let fullPath: string;
+    let fileName = path.join(__dirname, '../example/datatypes.dmodel');
```

Comment:
Generating this example is quite boring because it does not contain any entities, so no files are generated.

Reply:
The other examples can't be generated ATM because of #235. But we could change `qualified-names.dmodel` so it does not use any definitions from `datatypes.dmodel`, then it should work.

Reply:
For now I changed qualified-names.dmodel by removing the reference to datatypes.dmodel and use this one now.
So I can test parts of the generated structure. I hope that is enough :-)